site2/docs/transaction-api.md

```diff
@@ -0,0 +1,149 @@
+---
+id: transaction-api
+title: Transaction API(Developer Preview)
+sidebar_label: Transaction API
+---
+
+All messages in a transaction is available only to consumers after the transaction is committed. If a transaction is aborted, all the writes and acknowledgments in this transaction rollback. 
+
+Currently, Pulsar transaction is a developer preview feature. It is disabled by default. You can enable the feature and use transactions in your application in development environment.
+
+## Prerequisites
+1. To enable transactions in Pulsar, you need to configure the parameter in the `broker.conf` file.
+
+```
+transactionCoordinatorEnabled=true
+```
+
+2. Initialize transaction coordinator metadata, so the transaction coordinators can leverage advantages of the partitioned topic, such as load balance.
+
+```
+bin/pulsar initialize-transaction-coordinator-metadata -cs 127.0.0.1:2181 -c standalone
+```
+
+After initializing transaction coordinator metadata, you can use the transactions API. The following APIs are available.
+
+## Initialize Pulsar client 
+
+You can enable transaction for transaction client and initialize transaction coordinator client.
+
+```
+PulsarClient pulsarClient = PulsarClient.builder()
+        .serviceUrl("pulsar://localhost:6650")
+        .enableTransaction(true)
```

Comment:
@codelipenghui @sijie @jiazhai @rdhabalia  
is this a typo ?
I suggest to change this to "enableTransactions", I feel it sounds better in English language

Reply:
@codelipenghui how do you think about it?